dart/neural/ConstrainedGroupGradientMatrices.cpp

```diff
@@ -1512,8 +1550,10 @@ ConstrainedGroupGradientMatrices::getJacobianOfLCPOffsetClampingSubset(
   Eigen::MatrixXs dC = getJacobianOfC(world, wrt);
   if (wrt == WithRespectTo::VELOCITY)
   {
+    Eigen::MatrixXs ddamp = getDampingDiagonal(world);
     return getBounceDiagonals().asDiagonal() * -A_c.transpose()
-           * (Eigen::MatrixXs::Identity(mNumDOFs, mNumDOFs) - dt * Minv * dC);
+           * (Eigen::MatrixXs::Identity(mNumDOFs, mNumDOFs) - dt * Minv * dC
```

Comment:
probably more efficient to do `dt * Minv * (dC + ddamp)`

Reply:
Sure